db/seeds/prisons/TCI-thorn-cross.yml

```diff
@@ -22,14 +22,17 @@ recurring:
   fri:
   - 1345-1545
 unbookable:
-- 2018-12-25
-- 2018-12-26
-- 2019-01-01
+- 2019-12-24
+- 2019-12-25
+- 2019-12-26
+- 2019-12-31
+- 2020-01-01
+- 2020-01-02
+- 2020-01-03
```

Comment:
This slot (3/1) needs to be removed from the unbookable section as it's also under anomalous, which is causing a conflict/error